help-service/src/main/resources/application-test.yml

```
@@ -0,0 +1,27 @@
+      value-deserializer: org.apache.kafka.common.serialization.StringDeserializer
+      # value-deserializer: org.springframework.kafka.support.serializer.JsonDeserializer
+      auto-offset-reset: earliest
+  jpa:
+    show-sql: true
+    hibernate:
+      ddl-auto: create
+    properties:
+      hibernate:
+        spring.jpa.open-in-view: false
+        format_sql: true
+        default_batch_fetch_size: 10
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)